### PR TITLE
Added English Quotes

### PR DESCRIPTION
**Pull request by WhosChester**
### Description
Oh boy, let's try this again
 
1. Replaced the song lyrics I previously added with philosophical and movie quotes
2. Added more English quotes

### CODE REVIEW

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -39012,6 +39012,216 @@
...
+      "source": "Karl Marx",
+      "id": 6881,
+      "length": 147
+    },
+    {
+      "text": "As long as Man continues to be the ruthless destroyer or lower living beings, he will never know health or peace. For as long as men massacre animals, they will kill each other. Indeed, he who sows the seed of murder and pain cannot reap joy and love.",
+      "source": "Pythagoras",
+      "id": 6882,
+      "length": 251
+    },
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39027]: Remove

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -39012,6 +39012,216 @@
...
+    {
+      "text": "The fear of death follows from the fear of life. A man who lives fully is prepared to die at any time.",
+      "source": "Mark Twain",
+      "id": 6883,
+      "length": 102
+    },
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39033]: Probably remove

File: frontend/static/quotes/english.json [hunk 3]
```diff
@@ -39012,6 +39012,216 @@
...
+      "length": 104
+    },
+    {
+      "text": "You know you're in love when you can't fall asleep because reality is finally better than your dreams.",
+      "source": "Dr. Seuss",
+      "id": 6889,
+      "length": 102
+    },
+    {
+      "text": "I am Gunnery Sergeant Hartman, your senior drill instructor. From now on you will speak only when spoken to, and the first and last words out of your filthy sewers will be 'Sir'. Do you maggots understand that?",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39071]: Replace 'Sir' with \"Sir\"

File: frontend/static/quotes/english.json [hunk 4]
```diff
@@ -39012,6 +39012,216 @@
...
+      "length": 290
+    },
+    {
+      "text": "Maybe someday we could become friends. Friends who ride majestic, translucent steeds, shooting flaming arrows across the bridge of Hemdale.",
+      "source": "Will Ferrell, Step Brothers",
+      "id": 6892,
+      "length": 161
+    },
+    {
+      "text": "'Why doesn't it just do us in and get it over with? Christ, I don't know how much longer I can go on like this.' It was our one hundred and ninth year in the computer. He was speaking for all of us.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39089]: Replace ' with "

File: frontend/static/quotes/english.json [hunk 5]
```diff
@@ -39012,6 +39012,216 @@
...
+    {
+      "text": "I want to have the same last dream again. The one where I wake up, and I'm alive just as the four walls closed me within, my eyes are opened up with pure sunlight. I'm the first to know, my dearest friends, even if your hope has burned with time. Anything that's dead shall be regrown, and your vicious pain, your warning sign, you will be fine.",
+      "source": "Angels and Airwaves, The Adventure",
+      "id": 6898,
+      "length": 345
+    },
+    {
+      "text": "Look again at that dot. That's here. That's home. That's us. On it everyone you love, everyone you know, everyone you ever heard, every human being who ever was, lived out their lives. The aggregate of our joy and suffering, thousands of confident religions, ideologies, and economic doctrins, every hunter and forager, every hero and coward, every creator and destroyer of civilization, every king and peasant, every young couple in love, every mother and father, hopeful child, inventor and explorer, every teacher or morals, every corrupt politician, every superstar, ever supreme leader, every saint and sinner in the history of our species lived there-on a mote of dust suspended in a sunbeam.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39125]: Good quote

File: frontend/static/quotes/english.json [hunk 6]
```diff
@@ -39012,6 +39012,216 @@
...
+      "length": 213
+    },
+    {
+      "text": "Nothing in life is to be feared, it is only to be understood. Now is the time to understand more, so that we may fear less.",
+      "source": "Marie Curie",
+      "id": 6905,
+      "length": 123
+    },
+    {
+      "text": "I do not feel obliged to believe that the same God who has endowed us with sense, reason, and intellect has intended us to forgo their use.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39167]: Remove

File: frontend/static/quotes/english.json [hunk 7]
```diff
@@ -39012,6 +39012,216 @@
...
+      "length": 139
+    }, 
+    {
+      "text": "Rest is not idleness, and to lie sometimes on the grass under trees on a summer's day, listening to the murmur of the water, or watching the clouds float across the sky, is by no means a waste of time.",
+      "source": "John Lubbock, The Use Of Life",
+      "id": 6907,
+      "length": 201
+    },
+    {
+      "text": "Matter, though divisible in an extreme degree, is nevertheless not infinitely divisible. That is, there must be some point beyond which we cannot go in the division of matter.... I have chosen the word \"atom\" to signify these ultimate particles.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:39179]: Change `....` to `...`

**Comment by Miodec** [comment]
Please don't open new PRs. You can just commit to your master branch and they will automatically appear on this PR.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4890417027

**Comment by Miodec** [comment]
Length fields are incorrect

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4912706100

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4930822100

**Review by Miodec — APPROVED** [review]

**Pull request merged**